src/program/latex/mod.rs

```diff
@@ -0,0 +1,1600 @@
+        } else if text == "beta" {
+            Symbol::Beta
+        } else if text == "gamma" {
+            Symbol::Gamma
+        } else if text == "phi" {
+            Symbol::Phi
+        } else if text == "pi" {
+            Symbol::Pi
+        } else {
+            Symbol::Text(text)
+        }
+    }
+    /// Returns the LaTeX String for a given Symbol variant.
+    ///
+    /// # Arguments
+    /// `symbol` - A Symbol variant.
+    ///
+    /// # Examples
+    /// ```
+    /// use quil_rs::program::latex::{Parameter, Symbol};
+    /// let alpha = Symbol::get_symbol(
+    ///     &Parameter::Symbol(Symbol::Alpha)
+    /// );
+    /// ```
+    pub fn get_symbol(symbol: &Parameter) -> String {
+        match symbol {
+            Parameter::Symbol(Symbol::Alpha) => r"\alpha".to_string(),
+            Parameter::Symbol(Symbol::Beta) => r"\beta".to_string(),
+            Parameter::Symbol(Symbol::Gamma) => r"\gamma".to_string(),
+            Parameter::Symbol(Symbol::Phi) => r"\phi".to_string(),
+            Parameter::Symbol(Symbol::Pi) => r"\pi".to_string(),
+            Parameter::Symbol(Symbol::Text(text)) => format(format_args!(r#"\text{{{text}}}"#)),
+        }
+    }
+}
+
+/// Settings contains the metadata that allows the user to customize how the
+/// circuit is rendered or use the default implementation.
+#[derive(Debug)]
+pub struct Settings {
+    /// Convert numerical constants, e.g. pi, to LaTeX form.
+    pub texify_numerical_constants: bool,
+    /// Include all qubits implicitly referenced in the Quil program.
+    pub impute_missing_qubits: bool,
+    /// Label qubit lines.
+    pub label_qubit_lines: bool,
+    /// Write controlled rotations in compact form.
+    pub abbreviate_controlled_rotations: bool,
+    /// Extend the length of open wires at the right of the diagram.
+    pub qubit_line_open_wire_length: u32,
+    /// Align measurement operations to appear at the end of the diagram.
+    pub right_align_terminal_measurements: bool,
+}
+
+impl Default for Settings {
+    /// Returns the default Settings.
+    fn default() -> Self {
+        Self {
+            /// false: pi is π.
+            texify_numerical_constants: true,
+            /// true: `CNOT 0 2` would have three qubit lines: 0, 1, 2.
+            impute_missing_qubits: false,
+            /// false: remove Lstick/Rstick from latex.
+            label_qubit_lines: true,
+            /// true: `RX(pi)` displayed as `X_{\\pi}` instead of `R_X(\\pi)`.
+            abbreviate_controlled_rotations: false,
+            /// 0: condenses the size of subdiagrams.
+            qubit_line_open_wire_length: 1,
+            /// false: include Meter in the current column.
+            right_align_terminal_measurements: true,
+        }
+    }
+}
+
+impl Settings {
+    /// Returns a label as the qubit name to the left side of the wire.
+    ///
+    ///  # Arguments
+    /// `name` - name of the qubit
+    pub fn label_qubit_lines(&self, name: u64) -> String {
```

Comment:
This seems unrelated to the intent of `Settings`

Reply:
My intent with `Settings` is that if the user decides to use other than `default`, the Settings impl block will contain the functionality to apply those changes. It's just a way to group these effects into a single cohesive location.